gemset.nix

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
This is generated from the Gemfile lock to support the rest of the build.  Anyone who doesn't know what this is can ignore it.